src/test/java/com/savvato/tribeapp/services/AbstractServiceImplTest.java

```diff
@@ -24,6 +23,10 @@ public abstract class AbstractServiceImplTest {
     public static String USER1_NAME = "Fake A. Admin";
     public static String USER2_NAME = "Fake R. User"; // the R stand for Regular
 
+    // Phrase1 and matching word ids
+    public static long WORD_ID_PHRASE1 = 100L;
```

Comment:
Rename this for clarity. WORD_TABLE_INITIAL_ID comes to mind, but it of course doesn't necessarily have to be that.